tools/lsp/server/compiler.toit

```diff
@@ -198,7 +198,8 @@ class Compiler:
             while true:
               line = reader.read_line
               if line == "*******************": break
-              msg += line
+              if msg != "": msg += "\n"
+              msg += "$line"
```

Comment:
I'd use a list of lines and join them at the end.

Reply:
done.